tensilelite/rocisa/tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+envlist = py38, py39, py310
+
+[testenv]
+deps =
+    cmake
+    -r{toxinidir}/requirements.txt
+    nanobind
+    pytest>=5.4.1
+setenv =
+    PYTHONPATH = {envdir}/rocisa/lib
+commands =
+    mkdir -p {envdir}/rocisa
+    sh -c "cmake -S . -B {envdir}/rocisa -DCMAKE_CXX_COMPILER=/opt/rocm/bin/amdclang++ -DCMAKE_PREFIX_PATH=$(python3 -m nanobind --cmake_dir)"
```

Comment:
Don't hard code rocm path.

Reply:
this is a fallback in case the env is not found

Reply:
Oh sorry, wrong file

Reply:
Solved